tests/test_events.py

```diff
@@ -135,6 +138,19 @@ def test_get(self):
         for replica in Replica:
             self._test_get(replica)
 
+    def test_events_paging(self):
+        for replica in Replica:
+            resp = self.app.get(f"/v1/events?replica={replica.name}")
+            self.assertIn(resp.status_code, [200, 206])
+            content_key = resp.headers.get("X-OpenAPI-Paginated-Content-Key")
+            self.assertEqual("event_streams", content_key)
+            results = resp.json()
+            for key in content_key.split("."):
+                results = results[key]
+            for result in results:
+                for event in replay_event_stream(result):
+                    pass
```

Comment:
Is there something to be added here? If we are testing that the paging works and that data is being passed back, the length of the results object could be checked.